pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -146,6 +146,30 @@ fi
 `
 )
 
+// OSDs on PVC using a certain fast storage class need to do some tuning
+var defaultTuneFastSettings = map[string]string{
+	"osd_op_num_threads_per_shard_ssd":        "2",          // Default value of osd_op_num_threads_per_shard for SSDs
```

Comment:
setting these flags with the _ssd suffix won't affect disks that aren't detected as rotational by ceph - it's not changing the behaviour at all, just restating the existing default.

To override that, you'd need to use the setting without the _ssd suffix, e.g. set osd_op_num_threads_per_shard = 2.  This takes precedence over the _hdd or _ssd variants of the setting.

Reply:
#6383 will resolve this. Thanks